src/Halcyon/Datasource/DbDatasource.php

```diff
@@ -372,52 +337,31 @@ public function delete(string $dirName, string $fileName, string $extension)
     }
 
     /**
-     * Return the last modified date of an object
-     *
-     * @param  string  $dirName
-     * @param  string  $fileName
-     * @param  string  $extension
-     * @return int
+     * @inheritDoc
      */
-    public function lastModified(string $dirName, string $fileName, string $extension)
+    public function lastModified(string $dirName, string $fileName, string $extension): ?int
     {
         try {
             return Carbon::parse($this->getQuery()
                     ->where('path', $this->makeFilePath($dirName, $fileName, $extension))
                     ->first()->updated_at)->timestamp;
-        }
-        catch (Exception $ex) {
+        } catch (Exception $ex) {
             return null;
         }
     }
 
     /**
-     * Generate a cache key unique to this datasource.
-     *
-     * @param  string  $name
-     * @return string
+     * @inheritDoc
      */
-    public function makeCacheKey($name = '')
-    {
-        return crc32($this->source . $name);
-    }
```

Comment:
Why was this method removed?

Reply:
The `makeCacheKey` method is being provided by the abstract class `Winter\Storm\Halycon\Datasource\Datasource`. While this implementation of the method does use `$this->source`, `the `getPathsCacheKey()` has the source as part of the cache key prefix anyway, so it should still be unique. Is there any other reason why the source was necessary here?